api/src/main/java/org/openmrs/module/clientregistry/api/ClientRegistryManager.java

```diff
@@ -82,4 +91,30 @@ public void disableClientRegistry() {
 		
 		isRunning.set(false);
 	}
+	
+	/**
+	 * Determine the appropriate PatientService class based off of the client registry transaction
+	 * type configuration
+	 * 
+	 * @return PatientService class corresponding to the appropriate transaction type supported by
+	 *         the client registry
+	 * @throws IllegalArgumentException if defined transaction type is unsupported
+	 */
+	public CRPatientService getPatientService() throws IllegalArgumentException {
+		try {
+			String transactionMethodGlobalProperty = clientRegistryConfig.getClientRegistryTransactionMethod().toUpperCase();
+			
+			switch (ClientRegistryTransactionType.valueOf(transactionMethodGlobalProperty)) {
+				case FHIR:
+					return fhirPatientService;
+				case HL7:
+					throw new IllegalArgumentException("HL7 transaction type is currently unsupported");
```

Comment:
Do we plan to support HL7 or other formats in the future? If not, can we just simply support FHIR? cc: @pmanko @smallgod @ebambo

Reply:
I would vote for FHIR only as any other transformation will reside outside of this module

Reply:
Okay, gonna file a ticket to address this.